get_forecast.py

```diff
@@ -80,13 +80,14 @@ def get_secret(sm_client):
     return secret
 
 
-def send_slack(slack_url, message):
+def send_slack(slack_url, code_block_format_message):
```

Comment:
Passing it is is a great idea but in that case the variable name does not make sense, this opera should be text_message as the caller will decide if it si code_block or something else